haystack/cli/README.md

```diff
@@ -13,32 +13,42 @@ This is the repository where we keep the code for the Haystack CLI.
 
 To contribute to the tutorials please check out our [Contributing Guidelines](./Contributing.md)
 
-## Available commands
+## Available CLI commands
 
 ### `haystack prompt fetch`
 
-```
-Usage: haystack prompt fetch [OPTIONS] [PROMPT_NAME]...
+**Usage:**
 
-  Downloads a prompt from the official Haystack PromptHub and saves it locally
-  to ease use in environments with no network.
+```
+haystack prompt fetch [OPTIONS] [PROMPT_NAME]
+```
 
-  PROMPT_NAME can be specified multiple times.
+Download a prompt from the official [Haystack PromptHub](https://prompthub.deepset.ai/) and save it locally
+for easier use in environments with no network.
 
-  PROMPTHUB_CACHE_PATH environment variable can be set to change the default
-  folder in which the prompts will be saved in.
+- PROMPT_NAME can be specified multiple times.
 
-  If a custom PROMPTHUB_CACHE_PATH is used, remember to also use it for
-  Haystack invocations.
+- PROMPTHUB_CACHE_PATH environment variable can be set to change the default
+folder in which the prompts will be saved in. You can find the default cache path on your machine by running the following code:
 
-  The Haystack PromptHub is https://prompthub.deepset.ai/
+  ``` python
+  from platformdirs import user_data_dir
+  from pathlib import Path
+  Path(user_data_dir("haystack", "deepset")) / "prompthub_cache"
+  ```
 
-Options:
-  --help  Show this message and exit.
-```
+  - If a custom PROMPTHUB_CACHE_PATH is used, remember to also use it for Haystack invocations.
 
-Example usage:
+**Example:**
 
 ```
 haystack prompt fetch deepset/conversational-agent-with-tools deepset/summarization
 ```
+
+**Options:**
+
+`--help`  Show options and exit.
+
+### `--version`
```

Comment:
For consistency with `haystack prompt fetch`, this should read `haystack --version`

```suggestion
### `haystack --version`
```

Reply:
Hey @ZanSara, this might be a bit unrelated, but I was wondering why the command name is `haystack prompt fetch` but not `haystack prompt-fetch`. There's no command as `haystack prompt` right now. Do we maybe plan to add some function to it in the future?

Reply:
That's the idea, yes. Having it like this allows us to add more prompt-related features later (idk, upload, validation, whatever really)